### DISCUSSION TITLE
Removing sources

### DISCUSSION THREAD

**Discussion by tward1505** [lncrawl/lightnovel-crawler#2089] [Q&A] [answered]
Is it possible for me to remove or block sources? There are several I make a point of avoiding for one reason or another. And also thought that the search would go a bit faster if I trimmed down the source list.

I would prefer blocking it in such a way that unblocking is easy, but I guess updating or reinstalling would work fine.

And from searching I found in the Changing sources a reference to the "site-packages\sources\" folder which is my start, but now I'd like to know if there is a clean method to disabling the sources rather than my "bull in a china shop" first instinct.

**Comment by dipu-bd** [comment, marked as answer]
you can filter out by domain names for searching.

```
-x [REGEX], --sources [REGEX]
                        Filter out the sources to search for novels.
```
